gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/impl/AbstractConnectorPluginService.java

```diff
@@ -76,6 +78,18 @@ protected ConnectorPluginEntity convert(Plugin plugin) {
                     .collect(Collectors.toSet())
             );
         }
+        if (
+            plugin.manifest().properties() != null &&
+            plugin.manifest().properties().get("features") != null &&
+            !plugin.manifest().properties().get("features").isEmpty()
+        ) {
+            entity.setAvailableFeatures(
+                Arrays
+                    .stream(plugin.manifest().properties().get("features").split(","))
+                    .map(ConnectorFeature::valueOf)
```

Comment:
I would implement valueOfLabel as new enum for v4.

Reply:
👌🏻